### PR TITLE
Update qube-workflow-ci-cd.groovy

### PR DESCRIPTION
**Pull request by prabhuinbarajan**

### CODE REVIEW

File: qube-workflows/jenkins/qube-workflow-ci-cd.groovy [hunk 1]
```diff
@@ -3,39 +3,60 @@ import java.nio.file.*
...
+  def keys_loc=getValueFromConfig(config_file,"security.keys_loc")
+  def skipOverlay=!isKeyPresentInJSON(config_file,"security.keys_loc")
+  def skipBuild=getBoolValueFromConfig(config_file,"build.skip")
+  def skipTest=getBoolValueFromConfig(config_file,"test.skip")
+  def skipBake=getBoolValueFromConfig(config_file,"bake.skip")
+  def registryPrefix=""
+  if(!skipBake) {
+      registryPrefix=getValueFromConfig(config_file,"bake.registry_prefix")
+  }
+  def deploymentModeQA = getValueFromConfig(config_file,"deployment.environments.qa.deploy_type")
```
Comment (qubehyunji) [inline qube-workflows/jenkins/qube-workflow-ci-cd.groovy]: are these variables prefixed with 'deployment' going to be used later? Right now the only place using them is sprintf() function

Comment (prabhuinbarajan) [inline qube-workflows/jenkins/qube-workflow-ci-cd.groovy]: Yes, the deployment templating is not done yet. just the yaml / json framework is ready. we now need to use this within the workflow

Comment (prabhuinbarajan) [inline qube-workflows/jenkins/qube-workflow-ci-cd.groovy]: which will be a separate pull request

**Comment by qubehyunji** [comment]
:+1:
